Tests/BraveWalletTests/AccountActivityStoreTests.swift

```diff
@@ -76,6 +76,9 @@ class AccountActivityStoreTests: XCTestCase {
       // spd token, sol nft balance
       completion(mockSplTokenBalances[tokenMintAddress] ?? "", UInt8(0), mockSplTokenBalances[tokenMintAddress] ?? "", .success, "")
     }
+    rpcService._erc721Metadata = { _, _, _, completion in
+      completion("", .internalError, "")
+    }
```

Comment:
Can we have at least 1 test in each of these stores unit tests (`AccountActivityStoreTests `, `PortfolioStoreTests`, `SendTokenStoreTests`, `UserAssetStoresTests`) to verify this data is actually fetched?

Reply:
fixed in [00c5f80](https://github.com/brave/brave-ios/pull/6534/commits/00c5f80308e952e4dc6914ae61d3bbcddda7d726)